app/serializers/export/contact_serializer.rb

```diff
@@ -1,12 +1,28 @@
 module Export
   class ContactSerializer < ActiveModel::Serializer
     attributes :sharing,
-               :receiving,
+               :following,
```

Comment:
Yes, actually we need some mechanism to sync data between pods, since it is currenlty possible to have non-synchronized contact states accross pods. If one pod misses `Contact` message and then sends `Contact`  message itself for the same users , then this pod will see only `sharing` with `receiving` false, while actually it is `receiving` also. What do you think, maybe we should implement fetching for `Contact`s instead? So that it was possible to update `Contact` state on demand.

Also imagine that we import old archive, and `Contact` state has changed in the federation. Then it is possible that we have `sharing=true`, `receiving=false` in the archive, when the remote pod has `sharing=false`, `receiving=false`. Just setting  `sharing` to `true` would mean that we make the state non-synchronized. So generally we should request all the contacts for the migrating user, not only for `receiving=true`. But also it means that we can't just create them as is in case of mismatch. Because if user has deleted the contact but imports the old archive it doesn't mean he wants to create the contact again. So we should make some UI, that will report the user that there are some mismatches between you and the remote contact and let the user decide what to do: add contact or remove it again (and resend `Contact` if the state has changed). 

Also, since it is possible, that this `Contact` state mismatch happens during the normal operation, not only the migration, we allow that procedure to be called on demand (at least by a podmin). Therefore I think that fetching of `Contacts` from the remote pod for this purposes is a better way than using push model.